### PR TITLE
coded

### PR DESCRIPTION
**Pull request by Szpytowski**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -1 +1,260 @@
...
+function isThisBoolean(value) {
+    return value === false || value === true;
+}
+
+console.log(isThisBoolean(true));
+console.log(isThisBoolean(false));
+console.log(isThisBoolean('true'));
+
+// 4
+function getCircleArea(radius){
```
Comment (Solnick) [inline js/app.js]:
```suggestion
function getCircleArea(radius) {
```

File: js/app.js [hunk 2]
```diff
@@ -1 +1,260 @@
...
+
+const radius = 5;
+const area = getCircleArea(radius);
+console.log(area); // 78.53981633974483
+
+// 5
+function getCirclesAreaSum(radius1, radius2) {
+    const area1 = getCircleArea(radius1);
+    const area2 = getCircleArea(radius2);
+
```
Comment (Solnick) [inline js/app.js:38]: Don't leave more than one new line sign between code lines. One is enough to make the code more pleasant to read, 2 new lines is too many.

File: js/app.js [hunk 3]
```diff
@@ -1 +1,260 @@
...
+
+    return area1 + area2;
+}
+
+const circlesAreaSum = getCirclesAreaSum(5, 10);
+console.log(circlesAreaSum); // 392.69908169872417
+
+// 6
+const quarterOf = (month) => {
```
Comment (Solnick) [inline js/app.js:46]: How did you come up with the `=>` notation?

File: js/app.js [hunk 4]
```diff
@@ -1 +1,260 @@
...
+    return Math.ceil(month / 3);
+}
+
+// 7
+function convertMinutesToHours(minutes) {
+    return minutes / 60
```
Comment (Solnick) [inline js/app.js]: remember about `;`

File: js/app.js [hunk 5]
```diff
@@ -1 +1,260 @@
...
+console.log(getBillboardPrice('To be, or not to be', 20));  // 380, since there are 19 characters
+
+// 11
+function opposite(number) {
+    return (number * -1)
+}
+
+// 12
+function makeNegative(num) {
+    if (num >0) {
```
Comment (Solnick) [inline js/app.js]:
```suggestion
    if (num > 0) {
```

File: js/app.js [hunk 6]
```diff
@@ -1 +1,260 @@
...
+        return (num * -1)
+    }
+}
+
+// 13
+function removeChar(str){
+    return str.slice(1, -1);
```
Comment (Solnick) [inline js/app.js:94]: How did you found this `-1` solution?

File: js/app.js [hunk 7]
```diff
@@ -1 +1,260 @@
...
+};
+
+// 14
+function basicOp(operation, value1, value2){
```
Comment (Solnick) [inline js/app.js]: `value` is not a descriptive argument name. Please use better naming, it should be self-explenatory

File: js/app.js [hunk 8]
```diff
@@ -1 +1,260 @@
...
+    } else if (operation === '/') {
+        return value1 / value2;
+    } else {
+        return 0;
+    }
+}
+
+// 15
+function century(year) {
+    const century = Math.ceil (year / 100);
```
Comment (Solnick) [inline js/app.js]:
```suggestion
    const century = Math.ceil(year / 100);
```

File: js/app.js [hunk 9]
```diff
@@ -1 +1,260 @@
...
+    const firstLetterIsR = name.charAt(0).toLowerCase() === 'r';
+    if (firstLetterIsR) {
+        return name + " plays banjo";
+    } else {
+        return name + " does not play banjo";
+    }
+}
+
+// 18
+function rockPaperScissors(player1, player2) {
```
Comment (Solnick) [inline js/app.js]: player is not a descriptive name. Maybe player1Move?

File: js/app.js [hunk 10]
```diff
@@ -1 +1,260 @@
...
+        return 0;
+    }
+
+    if (
+        (player1 === 'rock' && player2 === 'scissors') ||
+        (player1 === 'scissors' && player2 === 'paper') ||
+        (player1 === 'paper' && player2 === 'rock')
+    ) {
+        return 1;
+    } else {
```
Comment (Solnick) [inline js/app.js]: do you remember what to do when there is the else statement with the return statement?

File: js/app.js [hunk 11]
```diff
@@ -1 +1,260 @@
...
+    if (calculationType === '+') {
+        return firstNumber + secondNumber;
+    } else if (calculationType === '-') {
+        return firstNumber - secondNumber;
+    } else if (calculationType === '*') {
+        return firstNumber * secondNumber;
+    } else if (calculationType === '/') {
+        return firstNumber / secondNumber;
+    } else {
+        return "null";
```
Comment (Solnick) [inline js/app.js]: `null` is an actual valid JS value, you don't need to wrap it as a string.

**Review by Solnick — CHANGES_REQUESTED** [review]

File: js/app.js (hunk 3 above)
Comment (Szpytowski) [reply js/app.js:46]: The solutions I had weren't working, so I googled it how to calculate quarter of the month in js. And found that on multiple examples they were using the => notation 
https://stackoverflow.com/questions/74586033/how-can-i-find-what-quarter-of-the-year-a-month-is-in-based-on-the-number-the-mo

File: js/app.js (hunk 7 above)
Comment (Szpytowski) [reply js/app.js]: Examples(Operator, value1, value2) --> output 
I copied it from the assignment. I will call it first and second number

**Review by Solnick — CHANGES_REQUESTED** [review]
Please adjust the code according to this comment https://github.com/Szpytowski/Javascript-functions/pull/1#discussion_r1765541360

**Review by Solnick — APPROVED** [review]

**Pull request merged**